crates/common/src/fuzz/mod.rs

```diff
@@ -160,24 +169,35 @@ impl FuzzRunner {
                 final_bindings.push(Rebind(elided_binding.to_string(), hex));
             }
 
-            final_bindings.extend(scenario_bindings.clone());
+            let handle = tokio::spawn(async move {
+                final_bindings.extend(scenario_bindings.clone());
+
+                let rainlang_string = RainDocument::compose_text(
+                    &dotrain,
+                    &ORDERBOOK_ORDER_ENTRYPOINTS,
+                    None,
+                    Some(final_bindings),
+                )
+                .unwrap();
+
+                let args = ForkEvalArgs {
+                    rainlang_string,
+                    source_index: 0,
+                    deployer: deployer.address,
+                    namespace: FullyQualifiedNamespace::default(),
+                    context: vec![],
+                    decode_errors: false,
+                };
+                fork_clone.fork_eval(args).await.unwrap()
```

Comment:
these unwraps in these spawns will panic threads, can we handle results properly here?

Reply:
done